capstone/src/main/webapp/index.html

```diff
@@ -2,30 +2,30 @@
 <!doctype html>
 <html lang="en">
   <head>
-        <meta charset="utf-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-        <link href="style.css" type="text/css" rel="stylesheet">
-        <title>Home</title>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
+    <link href="style.css" type="text/css" rel="stylesheet">
+    <title>Home</title>
   </head>
   <body class="text-center">
-        <div class="welcome">
-            <h1 class="cover-heading">Welcome.</h1>
-            <p class="lead">
-                This website will give you election Information
-                relevant to you, and allow you add election dates to your 
-                Google Calendar.
-            </p>
-        </div>
+    <div class="welcome">
+        <h1 class="cover-heading">Welcome.</h1>
```

Comment:
If you look at the linter, it specifies lines with errors. Line 12 is one of them.
It's indented 4 spaces more than the line above, when it should only be 1-3 spaces. (One way to fix it is just to make it even with the line above, then hit tab)